src/python/pants/backend/python/rules/python_test_runner.py

```diff
@@ -81,15 +115,33 @@ class TestTargetSetup:
 
   chrooted_sources = await Get[ChrootedPythonSources](HydratedTargets(all_targets))
 
-  merged_input_files = await Get[Digest](
+  coveragerc_digest = await Get[Digest](InputFilesContent, get_coveragerc_input(DEFAULT_COVERAGE_CONFIG.encode()))
+
+  merged_input_files: Digest = await Get[Digest](
     DirectoriesToMerge(
-      directories=(chrooted_sources.digest, resolved_requirements_pex.directory_digest)
+      directories=(
+        chrooted_sources.digest,
+        resolved_requirements_pex.directory_digest,
+        coveragerc_digest,
+      )
     ),
   )
+  test_target_sources_file_names = source_root_stripped_test_target_sources.snapshot.files
+  coverage_args = []
+  if test_options.values.run_coverage:
+    packages_to_cover = get_packages_to_cover(
+      test_target,
+      source_root_stripped_file_paths=test_target_sources_file_names,
+    )
+    coverage_args = [
+      '--cov-report=', # To not generate any output. https://pytest-cov.readthedocs.io/en/latest/config.html
+    ]
```

Comment:
Instead, if we don't have a `.coveragerc`, can we leave off this option?

Reply:
This line does not get executed if we haven't passed `--coverage` (see line 131)